Module.yaml

```diff
@@ -0,0 +1,62 @@
+version: 1.0.0
+name: OPCUADiscovery
+description: OPC UA Discovery looks for all available nodes onn OPC UA Server and forwards the payload to next module.
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_OPCUA_input.png
+type: process
+tags:
+  - OPC UA Discovery
+  - process
+categories:
+  - id: category
+    name: Process
```

Comment:
We don't have a category Process. I would suggest Policy category.

Reply:
updated. why do we have input, output and this one is called policy ?